DebugEngine/MagoNatDE/Engine.cpp

```diff
@@ -501,6 +501,8 @@ namespace Mago
         info.StdOutput = (HANDLE) hStdOutput;
         info.StdError = (HANDLE) hStdError;
         info.Suspend = true;
+		if (pszOptions && std::wstring(pszOptions) == L"--external-console")
+			info.NewConsole = true;

```

Comment:
There's a mixture of tabs and spaces here. This causes bad indentation.

Wouldn't it be better to add this to the launch flags?